lib/light.js

```diff
@@ -535,7 +535,7 @@ Object.assign(Light.prototype, within);
 
 
 /* istanbul ignore else */
-if (!!process.env.IS_TEST_MODE) {
+if (process.env.IS_TEST_MODE) {
```

Comment:
I'm guessing there is some coercion reasoning behind the !! Do you know what it is/was?

Reply:
Seems like something I might do for a reason, but I can't remember, so throw it out.